source/code/plugins/change_tracking_runner.rb

```diff
@@ -0,0 +1,105 @@
+require "rexml/document"
+require "cgi"
+require 'logger'
+require 'digest'
+require 'json'
+
+require_relative 'changetracking_lib'
+require_relative 'oms_common'
+
+
+class ChangeTrackingRunner 
+	CHANGE_TRACKING_FILE = "/etc/opt/omi/conf/omsconfig/configuration/ChangeTrackingInventory.xml"
```

Comment:
I'll test with the newer path - /var/opt/Microsoft/omsagent/tmp/changetrackinginventory.xml